src/electionguard/logs.py

```diff
@@ -16,12 +16,17 @@ class ElectionGuardLog(Singleton):
     """
 
     __logger: logging.Logger
+    __stream_handler: logging.StreamHandler
 
     def __init__(self) -> None:
         super().__init__()
 
         self.__logger = logging.getLogger("electionguard")
-        self.__logger.addHandler(get_stream_handler())
+        # Pythong's logger will use the most restrictive of the logger level and the handler level,
+        #   so set the logger to the lowest level the handler ever might log at
+        self.__logger.setLevel(logging.DEBUG)
```

Comment:
EG used to only log ERROR and above even though it was set at INFO, because logger has a level at both the logger and handler levels, it uses the most restrictive of those two, and if the logger isn't set it will use ERROR.